src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = sourceString
+    .split(';')
+    .map((styles) => styles.trim())
+    .filter((styles) => styles.length > 0)
+    .map((styles) => styles.split(':'))
+    .reduce(
+      (object, styles) => ({
+        ...object,
+        [styles[0].trim()]: styles[1].trim(),
+      }),
+      {},
+    );
+
+  return stylesObject;
```

Comment:
The function correctly splits the string by semicolons, trims whitespace, filters out empty styles, and reduces the array into an object. However, it would be nice to handle cases with multiple semicolons without any styles between them, although it's not required.